app/jobs/webpage_cache_job.rb

```diff
@@ -5,6 +5,7 @@ class WebpageCacheJob < ApplicationJob
   attr_reader :bookmark
 
   def perform bookmark:
-    WebpageCacheService::Cache.new(webpage: bookmark.webpage).exec
+    service = WebpageCacheService::Cache.new(webpage: bookmark.webpage).exec
+    bookmark.offline_caches << service.offline_cache
```

Comment:
turns out ... this was the whole cause of this PR

Reply:
you uh... have to save the data

Reply:
yeaaaaaaaaaaaaaaaah :/